requirements.txt

```diff
@@ -20,5 +19,6 @@ pyyaml
 redis
 requests==2.2.1
 sqlalchemy
+oslo.utils==1.9.0
```

Comment:
I assume it was simply moved from above ;) This change is not needed though, so I would recommend reverting it